### PR TITLE
[#4] Add Docker

### LINKED ISSUE #4: Dockerize this project

### PR DESCRIPTION
**Pull request by Jean-Beru**
close #4

### CODE REVIEW

File: .docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM node:10-alpine
+
+ENV WORKDIR=/app
+
+RUN apk add --update ffmpeg && \
+    rm -rf /var/cache/apk/* && \
+    mkdir $WORKDIR && \
+    cd $WORKDIR
```
Comment (kevin-verschaeve) [inline .docker/Dockerfile]: IMO these 2 lines are useless as Docker will create the directory for you with the `WORKDIR` instruction just below

see docs https://docs.docker.com/engine/reference/builder/#workdir

File: README.md [hunk 2]
```diff
@@ -1,11 +1,36 @@
...
+### Requirement
+
+NodeJS >= 10
+ 
+### Install application
 
 Run `yarn install`.
 
-## Start application
+###Start application
```
Comment (kevin-verschaeve) [inline README.md]: Why ? It breaks the readme rendering

Comment (kevin-verschaeve) [reply README.md]: Oups, I mean : "leave a space" :smile:

Comment (Jean-Beru) [reply README.md]: `###` instead of `##` to add a level. No ` ` because... it's a typo :smile:

File: .docker/Dockerfile (hunk 1 above)
Comment (Jean-Beru) [reply .docker/Dockerfile]: Didn't know. :+1:

**Review by kevin-verschaeve — APPROVED** [review]
:clap: :clap:

**Pull request merged**